scripts/wrappers/common/cluster/utils.py

```diff
@@ -13,17 +13,6 @@
 import socket
 
 
-def get_group():
```

Comment:
let's ensure this does not break https://github.com/ubuntu/microk8s/blob/master/scripts/wrappers/addons.py#L179